lib/cfg.js

```diff
@@ -93,38 +92,30 @@ const gccX86 = {
     isJmpInstruction: x => x.trim()[0] === 'j',
 };
 
-const clangX86 = {
+const clang = {
     filterData: asmArr => {
         const jmpLabelRegex = /\.LBB\d+_\d+:/;
         const isCode = x => x && x.text && (x.source !== null || jmpLabelRegex.test(x.text) || isFunctionName(x));
 
-        const removeComments = x => {
-            const pos = x.text.indexOf('#');
-            if (pos !== -1)
-                x.text = utils.trimRight(x.text.substring(0, pos));
-            return x;
-        };
-
         return _.chain(filterTextSection(asmArr))
             .map(_.clone)
             .filter(isCode)
-            .map(removeComments)
```

Comment:
Do we know what the comments that are meant to be stripped here actually looked like?

It is possible they were very long and made the graph difficult to read - so perhaps we should try to figure out what exactly the impact of this change would be.

Reply:
Clang seems to generate small comments sometimes, I don't believe they are too intrusive.
Example (x86):
```
square(int):                             # @square(int)
        pushq   %rbp
...
```
(arm64)
```
square(int):                             // @square(int)
        orr     w8, wzr, #0x1
...
```
Apparently we don't have a reliable way of detecting the target at the moment, so no way to determine what character to strip as comments. If there is a solution to this problem I'd be happy to apply it here.

Reply:
if nothing else, we can likely leave the comment stripping code in place by looking for a hash with a space after it, assuming literals never have a space between the hash and number (which I believe is true).
 `const pos = x.text.indexOf('# ');`

Reply:
Nice, I'll try this.